ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -100,11 +101,12 @@ class LocalAssemblerData
                                  sm.detJ * wp.getWeight();
 
             // Darcy velocity only computed for output.
-            auto const darcy_velocity = -(k * sm.dNdx *
```

Comment:
Good idea! ...and one more proof that auto const ... eval is not the best practice.